### PR TITLE
Functional tests - Update JS doc for FO pages

### PR DESCRIPTION
**Pull request by nesrineabdmouleh**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | 1.7.8.x
| Description?      | Update js doc for all FO pages
| Type?             | refacto
| Category?         | TE
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | no
| How to test?      | no need
| Possible impacts? | no

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/25158)

### CODE REVIEW

**Review by boubkerbribri — APPROVED** [review]

**Review by PierreRambaud — APPROVED** [review]

File: tests/UI/pages/FO/myAccount/creditSlips.js [hunk 1]
```diff
@@ -1,12 +1,21 @@
 require('module-alias/register');
 const FOBasePage = require('@pages/FO/FObasePage');
 
-class CreditSlips extends FOBasePage {
+/**
+ * Credit slip page, contains functions that can be used on the page
+ * @class
+ * @extends FOBasePage
+ */
+class CreditSlip extends FOBasePage {
```
Comment (Progi1984) [inline tests/UI/pages/FO/myAccount/creditSlips.js:9]: The file shouldn't be renamed ?

**Review by Progi1984 — COMMENTED** [review]
A small comment

**Review by matthieu-rolland — APPROVED** [review]

File: tests/UI/pages/FO/myAccount/creditSlips.js (hunk 1 above)
Comment (boubkerbribri) [reply tests/UI/pages/FO/myAccount/creditSlips.js:9]: if renamed, a lot of updates will have to be done. It will be on another PR.

**Comment by Progi1984** [comment]
## WAIT FOR 1780 RELEASE

**Comment by Progi1984** [comment]
Thanks @nesrineabdmouleh

**Pull request merged**